webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const HtmlPlugin = require('html-webpack-plugin');
+const CleanPlugin = require('clean-webpack-plugin');
+/* eslint-disable-next-line no-undef */
+module.exports = {
+  entry: './src/index.js', 
+  output: {
+    filename: 'bundle.[hash].js'
+  },
+  devServer: {
+    port: 7890
+  },
+  plugins: [
+    new HtmlPlugin({ template: './src/index.html' }),
+    new CleanPlugin('./dist')
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use:{
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react'],
+            cacheDirectory: true
+          }
+        }
+      },
+      {
+        test: /\.css$/,
```

Comment:
You're missing some loaders, you want this section to look like this: 
```suggestion
        {
        test: /\.css$/,
        use: [
          {
            loader: 'style-loader',
            options: { sourceMap: true }
          },
          {
            loader: 'css-loader',
            options: {
              sourceMap: true,
              modules: true,
              importLoaders: 1
            }
          },
          {
            loader: 'postcss-loader',
            options: {
              sourceMap: true,
              plugins: [
                require('autoprefixer')(),
                require('postcss-nested')()
              ]
            }
          }
        ]
      },
```